assets/js/atomic/components/product/button/index.js

```diff
@@ -125,8 +125,8 @@ const ProductButton = ( { product, className } ) => {
 			firstMount.current = false;
 			return;
 		}
-		// Test if we have our Event defined
-		if ( Object.entries( Event ).length !== 0 ) {
+		// In IE, Event is an object and can't be instantiated with `new Event()`.
+		if ( typeof Event === 'function' ) {
```

Comment:
Fix looks fine, but I wonder if we should just default the Event fallback (around line 84) to `null` instead of an empty object too?

Reply:
Sounds good. Done in e2007514d3513f89d85c863ba5d6c51a8e97288e.